### PR TITLE
Updated the installation guide

### PR DESCRIPTION
**Pull request by maxomnia**

### CODE REVIEW

**Comment by maxomnia** [comment]
# Don't merge yet

File: README.md [hunk 1]
```diff
@@ -19,19 +25,19 @@ npm run certificate
 npm run start
 ```
```
Comment (maximiliani) [inline README.md:26]: This bash belongs to the installation!

File: README.md [hunk 2]
```diff
@@ -19,19 +25,19 @@ npm run certificate
...
 
-### Relaunching the application
+## Relaunching the application
 To relaunch the "das kalender" application, you will need to run the following commands
 ```bash
 cd <your_path>
 npm start
 ```
-<your_path> needs to be replaced with the main directory titled something like *C:\\...\pm_project_rapla*
+<your_path> needs to be replaced with the main directory of the application titled something like *C:\\...\pm_project_rapla*
```
Comment (maximiliani) [inline README.md:36]: Please use linux paths. Windows is ugly... An alternative would be to say: "To relaunch the application, please run `npm start` inside the `pm_project_rapla` folder."

File: README.md [hunk 3]
```diff
@@ -19,19 +25,19 @@ npm run certificate
 npm run start
 ```
 
-Now you will be able to visit the web-based calendar UI under https://localhost:8080/
-
-The API documentation is available at https://localhost:8080/docs.
+Now you will be able to visit the web-based calendar interface at https://localhost:8080/
```
Comment (maximiliani) [inline README.md:29]: I think you should mention the API here, since it is a beautiful page ;)

**Review by maximiliani — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (maxomnia) [reply README.md:29]: Steht ganz unten. Ist ja für den Endnutzer nicht so relevant.

**Review by maximiliani — APPROVED** [review]

**Pull request merged**